cifar100 loaded
There are 40000 training samples and 10000 validation samples
There are 10000 test samples.
This is records for stage 3
Train with the egdis selected dataset.
Now try to run the algorithm EGDIS with the generated sample dataset.
Selected 12592 samples
Use multiple GPU to train the network.
load parameters
Traceback (most recent call last):
  File "cifar100_experiments.py", line 64, in <module>
    batch_size=batch_size, stage=args.stage)
  File "/home/s1572869/MSc_results/lib/experiments.py", line 311, in run_egdis
    dataset, batch_size=batch_size, name="egdis", stage=stage)
  File "/home/s1572869/MSc_results/lib/experiments.py", line 193, in train_with_original
    name + "size_" + str(len(train[1])) + "_stage_" + str(stage - 1) + "_set_ckpt.pth"))
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/site-packages/torch/serialization.py", line 525, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/site-packages/torch/serialization.py", line 212, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/s1572869/miniconda3/envs/mlp/lib/python3.7/site-packages/torch/serialization.py", line 193, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/home/s1572869/MSc_results/models/cifar100/egdissize_12592_stage_2_set_ckpt.pth'
